README.md

```diff
@@ -768,6 +768,21 @@ const Counter = flow(
 
 <!-- TODO: keep props separate by starting flow() with props => ({props}) -->
 
+## React Hooks Equivalents
+
+| React Hook                                                                                 | Ad-hok                                                  |
```

Comment:
Yikes, this looks insane in the GitHub diff, but in an editor without line-wrapping is reasonable to read.